invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -0,0 +1,286 @@
+            funder_string += f"award_identifiers_identifier={identifier}; "
+            funder_string += f"award_title={title}; "
+            funder_string += f"award_number={number}; "
+            funder_string += f"funder_id={id}; "
+            funder_string += f"funder_name={name}; "
+            funders.append(funder_string)
+
+        return funders
+
+    def last_updated(self, obj):
+        """Get date last updated."""
+        return [obj["updated"]]
+
+    def get_locations(self, obj):
+        """Get locations."""
+        locations = []
+
+        access_location = obj["metadata"].get("locations", {})
+        if access_location == {}:
+            return missing
+
+        for location in obj["metadata"].get("locations", []).get("features", []):
+            location_string = ""
+
+            place = location.get("place")
+            description = location.get("description")
+            if place:
+                location_string += f"name={place}; "
+            if description:
+                location_string += f"description={description}"
+            geometry = location.get("geometry")
+            if geometry:
+                geo_type = geometry["type"]
+                if geo_type == "Point":
+                    coords = geometry["coordinates"]
+                    location_string += f"; lat={coords[0]}; lon={coords[1]}"
+
+            locations.append(location_string)
+
+        return locations
+
+    def get_id_community(self, obj):
+        """Get community IDs."""
+        if "parent" in obj:
+            return obj["parent"]["communities"].get("ids", missing)
+        return missing
+
+    def get_id(self, obj):
+        """Get ID."""
+        if "parent" in obj:
+            return obj["parent"]["id"]
+        return missing
+
+    def get_titles(self, obj):
+        """Get titles."""
+        return [obj["metadata"]["title"]]
+
+    def get_identifiers(self, obj):
+        """Get identifiers."""
+        items = []
+        items.extend(i["identifier"] for i in obj["metadata"].get("identifiers", []))
+        items.extend(p["identifier"] for p in obj.get("pids", {}).values())
+
+        return items or missing
+
+    def get_creators(self, obj):
+        """Get creators."""
+        return [c["person_or_org"]["name"] for c in obj["metadata"].get("creators", [])]
+
+    def get_relations(self, obj):
+        """Get relations."""
+        rels = []
+
+        # Fundings
+        # FIXME: Add after UI support is there
+
+        # Alternate identifiers
+        for a in obj["metadata"].get("alternate_identifiers", []):
+            rels.append(f"{a['scheme']}:{a['identifier']}")
+
+        # Related identifiers
+        for a in obj["metadata"].get("related_identifiers", []):
+            rels.append(f"{a['scheme']}:{a['identifier']}")
+
+        return rels or missing
+
+    def get_rights(self, obj):
+        """Get rights."""
+        rights = []
+        access_right = obj.get("access", {}).get("status", {})
```

Comment:
isn't `status` a string, not a dict? So we would just expect a string value or None?
```suggestion
        access_right = obj.get("access", {}).get("status")
```